modules/adlooxAnalyticsAdapter.js

```diff
@@ -109,6 +109,10 @@ analyticsAdapter.enableAnalytics = function(config) {
     logError(MODULE, 'invalid js options value');
     return;
   }
+  if (isStr(config.options.js) && !/\.adlooxtracking\.(com|ru)$/.test(parseUrl(config.options.js, { 'noDecodeWholeURL': true }).host)) {
```

Comment:
what if TLD is same as window.location?

Reply:
Maybe better to amend `parseUrl` to use `new URL()` (instead of `<a/>`) now you have ~20 other users of this in Prebid.js already? You could then potentially move them over to this; or would you prefer to deprecate this function altogether and have the modules themselves just inline `new URL()`?

As for the consequences, for us, this might affect an internal test page, but nothing we would not just workaround or fixup with a latter PR.